### ISSUE TITLE
Add groups support

### ISSUE THREAD

**Issue by olevitt** [InseeFrLab/k8s-onboarding#50]
We should add basic (that is, considering group management is out of scope of this app) groups support.  
Features that I can think of :  
* Get `groups` list from user's token
* For each `group` in `groups`, allow user to onboard with a process similar to the already existing one (create namespace and give permissions in it to the group)  
* On cluster info page : make the user aware of the different namespaces he has access to (his own namespace + namespaces from the groups he belongs to)  

The OIDC proxy is already group-aware so everything should work well once the onboarding process is done

**Comment by micedre** [comment]
Are the groups created elsewhere or do you want to be able to create groups from the app? 

Is this really, create namespace from group, or the other way around : create a namespace and then the associated group (that all members can modify) ?

**Comment by olevitt** [comment]
For the time being, I was gonna limit the scope of this project by considering group management out of the scope of it.  
That means that `k8s-onboarding` would just let the user create a namespace and the associated rolebinding (giving permission to group instead of user) for each group present in the token.  
Group management (adding user to group, creating groups ...) would still need to be done one way or another in the identity provider.
